src/main/java/de/voidtech/gerald/commands/fun/LmgtfyCommand.java

```diff
@@ -0,0 +1,57 @@
+package main.java.de.voidtech.gerald.commands.fun;
+
+import main.java.de.voidtech.gerald.annotations.Command;
+import main.java.de.voidtech.gerald.commands.AbstractCommand;
+import main.java.de.voidtech.gerald.commands.CommandCategory;
+import net.dv8tion.jda.api.entities.Message;
+
+import java.util.List;
+
+@Command
+public class LmgtfyCommand extends AbstractCommand {
+
+    @SuppressWarnings("FieldCanBeLocal")
```

Comment:
Why is there a supress warning annotation?
Also since it's a constant that is attached to the class it should be `private static final String GOOGLE_QUERY`